src/main/java/com/vmtapp/enterprise/service/TicketServiceStub.java

```diff
@@ -36,7 +38,7 @@ public int checkUserRole(String email) {
     }
 
 
-
+    @Autowired
```

Comment:
Added @Autowired annotation to accomplish autowiring between of ITicketDao
https://docs.sonarqube.org/latest/user-guide/issues/

Reply:
Good.